jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/SliderTests.java

```diff
@@ -0,0 +1,221 @@
+package io.github.epam.angular.tests.elements.common;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.*;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+import static org.testng.Assert.*;
+
+public class SliderTests extends TestsInit {
```

Comment:
Split this class into unit and UI tests accroding to the [Angular_tests_guide.docx](https://epam-my.sharepoint.com/:w:/r/personal/alexey_girin_epam_com/_layouts/15/Doc.aspx?sourcedoc=%7BE3A93BE7-D0FF-45FD-B3D2-804A31B2482C%7D&file=Angular_tests_guide.docx&action=default&mobileredirect=true)